jaypeasee/sorting-hat-quiz
# Testing

#### What’s this PR do?
This PR completes testing for the current functionality of the application. There are 30 tests total - 29 unit tests and 1 integration test. 

Components tested:
1. `Error` - unit
2. `Intro` - unit
3. `Name` - unit
4. `Question` - unit
5. `Result` - unit (async)
6. `App` - integration (async)

#### Where should the reviewer start?
Review the tests in the order above.

#### How should this be manually tested?
1. Look to see if there are any holes in the testing - what else needs to be tested?
2. Run `npm test` to confirm each test is passing.

#### Any background context you want to provide?
This completes testing for the application (assuming I don't add any new content or functionality).

#### What are the relevant tickets?
* #22 
* #23 
* #27 
* #5 
* #4 
* #3 
* #2
